scripts/qesap/lib/config.py

```diff
@@ -239,6 +239,24 @@ def has_ansible(self):
         """
         return "ansible" in self.conf
 
+    def has_variable(self, variable_path):
```

Comment:
Not strictly needed but nice to have. 

I makes `has_ansible` superfluous. What about replacing any occurrence of `has_ansible` with `has_variable(['ansible'])` ?